.changeset/warm-toes-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+[LegacyCard] Fix updated print styles and spacing
```

Comment:
Sorry, have to add this changeset so I can cut a release and sneak this into the next web bump